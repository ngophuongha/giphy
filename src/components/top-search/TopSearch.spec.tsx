import { render } from "@testing-library/react";
import { TopSearch } from "./index";

describe("Top Search", () => {
  it("should render without crashing", () => {
    const { container } = render(<TopSearch />);
    expect(container).toMatchSnapshot();
  });
});
