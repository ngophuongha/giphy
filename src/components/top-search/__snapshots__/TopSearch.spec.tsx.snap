// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top Search should render without crashing 1`] = `
<div>
  <div
    class="my-4 input-group"
  >
    <input
      aria-label="gif-sticker-search"
      class="form-control"
      placeholder="Search all the GIFs and Stickers"
    />
    <span
      class="input-group-text"
      id="basic-addon2"
    >
      Search
    </span>
  </div>
</div>
`;
